vars/mtaBuild.groovy

```diff
@@ -166,6 +173,48 @@ void call(Map parameters = [:]) {
             $mtaCall
             """
 
+            echo "[INFO] Looping dockerBasedModules."
```

Comment:
Put `echo` and for loop in `if(configuration.dockerBasedModules){..}` to hide the output when this is not used.